lib/ramble/ramble/test/util/stats.py

```diff
@@ -0,0 +1,51 @@
+# Copyright 2022-2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+# https://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+# <LICENSE-MIT or https://opensource.org/licenses/MIT>, at your
+# option. This file may not be copied, modified, or distributed
+# except according to those terms.
+
+import ramble.util.stats
+
+
+def test_stats_for_repeat_foms():
+    test_values = [-2, 0, 2, 5.5]
+    test_units = 's'
+
+    expected_output = [(-2, 's', 'summary::min'),
```

Comment:
I'd prefer if this used `pytest.mark.parametrize` over an internal loop.

Reply:
I've rewritten this to test the stats classes directly with parametrize, and cut out the loops and the variations that aren't testing a unique result. Please let me know if this is what you were thinking and if you have any suggestions.